components/brave_adaptive_captcha/url_loader.h

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADAPTIVE_CAPTCHA_URL_LOADER_H_
+#define BRAVE_COMPONENTS_BRAVE_ADAPTIVE_CAPTCHA_URL_LOADER_H_
+
+#include <list>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+
+namespace content {
+class BrowserContext;
+}  // namespace content
+
+namespace brave_adaptive_captcha {
+
+class UrlLoader {
```

Comment:
you can probably reuse `brave/components/api_request_helper.h`

Reply:
Thanks for the tip on that! Done.